src/Symfony/Component/Form/ChoiceList/Loader/CallbackChoiceLoader.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\ChoiceList\Loader;
+
+use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
+
+/**
+ * Loads a {@link ArrayChoiceList} interface from any callable.
+ *
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+class CallbackChoiceLoader implements ChoiceLoaderInterface
+{
+    /**
+     * The callable used to load the choices.
+     *
+     * @var callable
+     */
+    private $callback;
+
+    /**
+     * The loaded choice list.
+     *
+     * @var ArrayChoiceList
+     */
+    private $choiceList;
+
```

Comment:
@fabpot Sorry, I don't understand where you want me to add and remove things.

Back then I had changed the first phpdoc from `The callable used to load the choices.` to `The callable used to load the array of choices.` thinking that both phpdocs would be less confusing.